concourse_ci/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+    "categories": [
+        "Collaboration"
+    ],
+    "guid": "054cc9fb-01c4-4f05-98b5-fae828746787",
+    "is_public": true,
+    "maintainer": "help@datadoghq.com",
+    "public_title": "Datadog-Concourse-ci Integration",
+    "support": "core",
+    "supported_os": [
+        "linux",
+        "mac_os",
+        "windows"
+      ],
+    "type": "check",
```

Comment:
I am not sure this is a "check" since the metrics are sent directly to the agent using Statsd.

Reply:
Its not, but this might be a first of its kind. Setting the type to `dogstatsd`, the only other integration that comes close is [ambassador](https://docs.datadoghq.com/integrations/ambassador/).

Reply:
The type is validated here: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/manifest.py#L315

We can extend the validation scheme to include `dogstatsd` if that works downstream (IDK what's using that field in the backend)